pyproject.toml

```diff
@@ -64,6 +64,7 @@ dependencies = [  # Specifier syntax: https://peps.python.org/pep-0631/
     "sqlalchemy >= 1.4.31",
     "watchdog >= 0.9.0",
     "typing-extensions >= 4.6.0; python_version < '3.12'",
+    "sphinx-click (>=6.0.0,<7.0.0)",
```

Comment:
Version 6.0.0 is currently the latest version of sphinx-click. So there is no known problem which would justify adding an [upper version constraint](https://iscinumpy.dev/post/bound-version-constraints/).